src/punctuation2.js

```diff
@@ -394,18 +419,12 @@ function checkRule(rule, field, subfield1, subfield2) {
     return false;
   }
 
-  //nvdebug(`${name}: ACCEPT ${rule.code}/${subfield1.code}, SF2=${rule.followedBy}/${subfield2 ? subfield2.code : '#'}`, debug);
+  nvdebug(`${rule.name ? rule.name : '<unnamed>'}: ACCEPT ${rule.code} (${subfield1.code}), SF2=${rule.followedBy} (${subfield2 ? subfield2.code : '#'})`, debug);
```

Comment:
comment?

Reply:
Punctuation rules can have 'name', which might make things easier to remember/debug, but 'name' is optional.

Reply:
Srry about bad articulation. Should this be commented out?

Reply:
Commented out